homework_1.css

```diff
@@ -11,52 +12,148 @@ body {
     text-align: center;
   }
 
-.wrapper div {
-    border: 1px solid black;
-}
-
 .wrapper > * {
     flex: 1 100%;
   }
 
+/***************/
+/* header */
 .header {
     font-size: 24px;
     color: rgb(107, 96, 96);
     padding-bottom: 20px;
 }
 
+/***************/
+/*left sidebar*/
 .sidebar_left {
     font-size: 20px;
     flex: 1;
     color: rgb(116, 110, 110);
-    text-align: center;
 }
 
-.sidebar_right {
-    font-size: 20px;
-    flex: 1;
-    text-align: center;
-    color: rgb(116, 110, 110);
+.sidebar_left label:last-child {
+    padding-bottom: 30px;
 }
 
-.input_radio {
+#padd {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 20px;
+}
+/*Text box*/
+input[type = text] {
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
     font-size: 16px;
-    text-align: left;
-    color: darkgrey;
+    opacity: 0.5;
+    height: 25px;
+    border-right: 1px solid grey;
```

Comment:
Define colors in a single way through all application (`#xxxxxx` format is preferred)